ios/AppLovinMAXAdViewUIComponent.h

```diff
@@ -7,14 +7,15 @@ NS_ASSUME_NONNULL_BEGIN
 @interface AppLovinMAXAdViewUIComponent : NSObject
 
 @property (nonatomic, strong, readonly) MAAdView *adView;
+@property (nonatomic, readonly) NSString *adUnitIdentifier;
```

Comment:
style nit with codebase, `copy` for strings
```suggestion
@property (nonatomic, copy, readonly) NSString *adUnitIdentifier;
```